pkg/storaged/client.js

```diff
@@ -1144,6 +1146,7 @@ function stratis2_fetch_manager_properties(proxy) {
                 proxy.StoppedPools[uuid] = {
                     devs: l.devs,
                     key_description: { t: "(bv)", v: [true, l.key_description] },
+                    clevis_info: { t: "(bv)", v: [true, l.clevis_info] },
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230531-100427-5b32f270-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)